Source/Atomic/UI/SystemUI/DebugHud.h

```diff
@@ -106,6 +106,21 @@ class ATOMIC_API DebugHud : public Object
     /// Clear all application-specific stats.
     void ClearAppStats();
 
+    //Atomic Begin
+
+    bool GetSceneOpen() { return isSceneOpen_; }
+    void SetSceneOpen(bool isOpen) { isSceneOpen_ = isOpen; }
+
+    String GetStatsString() { return statsString_; }
+    String GetModeString() { return modeString_; }
+    String GetProfilerString() { return profilerString_; }

```

Comment:
These call all return const String& otherwise duplicating strings on return, the members can also be marked const